src/compiler/node_generator.cc

```diff
@@ -74,8 +74,16 @@ grpc::string GetJSMessageFilename(const grpc::string& filename) {
 
 // Given a filename like foo/bar/baz.proto, returns the root directory
 // path ../../
-grpc::string GetRootPath(const grpc::string& filename) {
-  size_t slashes = std::count(filename.begin(), filename.end(), '/');
+grpc::string GetRootPath(const grpc::string& from_filename,
+                         const grpc::string& to_filename) {
+  if (to_filename.find("google/protobuf") == 0) {
+    // Well-known types (.proto files in the google/protobuf directory) are
+    // assumed to come from the 'google-protobuf' npm package.  We may want to
+    // generalize this exception later by letting others put generated code in
+    // their own npm packages.
+    return "google-protobuf/";
```

Comment:
Does this mean that for all .proto files that are in google/protobuf/ directory , we will expect their generated code to be part of google-protobuf npm package? That is true for well known protos, but it is not necessarily true for other .proto files. Should we only do this for .proto files that are actually supposed to be part of protobuf runtime?  (e.g. basically .proto files that are checked in google/protobuf github repo).